tools/export/mcuxpresso/empty_cproject.tmpl

```diff
@@ -0,0 +1,3 @@
+{% extends "mcuxpresso/.cproject.tmpl" %}
+
+{% block cpu_config %}{% endblock %}
```

Comment:
My understanding was that without a `<target>_cproject.tmpl` exporting was not supported. Therefore, your exporter would never be asked to export with this template.